### PR TITLE
[enocean] Improve incoming message handling for Eltako rollershutters

### PR DESCRIPTION
**Pull request by fruggy83**
Added the ability to handle PTM/RPS messages to rollershutter things and improved the calculation of rollershutter position. 

This PR solves #5702. 

Signed-off-by: Daniel Weber uni@fruggy.de

### CODE REVIEW

File: bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_3F/A5_3F_7F_EltakoFSB.java [hunk 1]
```diff
@@ -96,16 +96,24 @@ protected State convertToStateImpl(String channelId, String channelTypeId,
...
+            if (config != null) {
                 EnOceanChannelRollershutterConfig c = config.as(EnOceanChannelRollershutterConfig.class);
-                if (c.shutTime != -1 && c.shutTime != 0) {
-                    return new PercentType(Math.min(100, (Math.max(0, current.intValue()
-                            + direction * ((duration * PercentType.HUNDRED.intValue()) / c.shutTime)))));
+                if (duration == c.shutTime) {
+                    return getDB_1() == MoveUp ? PercentType.ZERO : PercentType.HUNDRED;
+                } else {
+                    PercentType current = PercentType.ZERO;
+                    if (currentState != UnDefType.UNDEF) {
```
Comment (J-N-K) [inline bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_3F/A5_3F_7F_EltakoFSB.java]:
```suggestion
                    if (currentState instanceof PercentType) {
```

Comment (fruggy83) [reply bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_3F/A5_3F_7F_EltakoFSB.java]: Good point, thanks for the hint. Fixed it.

**Comment by J-N-K** [comment]
Signed-off missing on zero-change-commits.

**Comment by J-N-K** [comment]
Jenkins OOM again...

**Pull request merged**